--- tool_follow.f90	2015-04-21 13:45:09.000000000 -0400
+++ swifter/tool/tool_follow.f90	2015-04-21 13:45:40.000000000 -0400
@@ -191,7 +191,7 @@
                               ic=ic+1
                               IF (MOD(ic,nskp) == 0) THEN
                                    WRITE(7,1001)t,ifol,xh,yh,zh,vxh,vyh,vzh
- 1001                              FORMAT(1x,e15.7,1x,i3,6(1x,es15.7))
+ 1001                              FORMAT(1x,e23.15,1x,i3,6(1x,es23.15))
                               END IF
                               tmax=t
                          END IF
